spiped/README

```diff
@@ -8,8 +8,8 @@ usage: spiped {-e | -d} -s <source socket> -t <target socket> -k <key file>
 
 Options:
     -c <source chmod>
-	The access permissions of the source socket (if it's a Unix socket).
```

Comment:
I think the original version was correct -- these lines should begin with a single `TAB` character.